driver-core/src/main/com/mongodb/ClientEncryptionSettings.java

```diff
@@ -23,17 +23,22 @@
 import static com.mongodb.assertions.Assertions.notNull;
 
 /**
- * The key vault encryption settings
+ * The client-side settings for data key creation and explicit encryption.
+ *
+ * <p>
+ * Explicit encryption/decryption is a community feature, enabled with the new {@code com.mongodb.client.vault.ClientEncryption} type,
+ * for which this is the settings.
+ * </p>
  *
  * @since 3.11
  */
-public final class KeyVaultEncryptionSettings {
+public final class ClientEncryptionSettings {
```

Comment:
Not sure what to call this class if AutoEncryptSettings is renamed to ClientSideEncryptionSettings

Reply:
Since AutoEncryptSettings name remains, this seems like the right name.